NewUser'sGuide.md

```diff
@@ -0,0 +1,250 @@
+- **Citra** - a Nintendo 3DS emulator.
+- **Colem** - a ColecoVision and Coleco Adam emulator.
+- **Desmume** - a Nintendo DS emulator.
+- **DGen** - a Sega Genesis/Megadrive emulator.
+- **Dolphin** - a GameCube and Nintendo Wii emulator.
+- **DOSBox** - an MS-DOS emulator.
+- **EasyRPG Player** - for RPG Maker 2000 and 2003 games.
+- **Frotz** - a Z-code emulator for text adventure games.
+- **FS-UAE** - an Amiga emulator.
+- **Hatari** - an Atari ST emulator.
+- **jzlntv** - an Intellivision emulator.
+- **Libretro** - a cross-platform front-end interface for emulation suites such as RetroArch.
+- **Linux** - for native Linux games and applications.
+- **MAME** - Multiple Arcade Machine Emulator for arcade games.
+- **Mednafen** - a multi-system emulator.
+- **melonDS** - a Nintendo DS emulator.
+- **MicroM8** - an Apple II emulator.
+- **Mupen64Plus** - a Nintendo 64 emulator.
+- **O2EM** - a Magnavox Odyssey 2 emulator.
+- **Osmose** - a Sega Master System emulator.
+- **PCem** - an IBM PC emulator.
+- **PCSX2** - a PlayStation 2 emulator.
+- **PICO-8** -  a fantasy (virtual) console.
+- **PPSSPP** - a Sony PSP emulator.
+- **Redream** - a Sega Dreamcast emulator.
+- **Reicast** - another Sega Dreamcast emulator.
+- **ResidualVM** - a 3D point-and-click adventure games engine.
+- **RPCS3** - a PlayStation 3 emulator.
+- **Ryujinx** - a Nintendo Switch emulator.
+- **ScummVM** - for 2D point-and-click adventure games.
+- **Snes9x** - a Super Nintendo emulator.
+- **Speccy** - a Sinclair ZX Spectrum emulator.
+- **Steam** - runs the Steam Linux client.
+- **Stella** - an Atari 2600 emulator.
+- **TIC-80** - a fantasy (virtual) computer for making, playing and sharing games with.
+- **Vice** - a Commodore emulator.
+- **Virtual Jaguar** - an Atari Jaguar emulator.
+- **Web** - runs web games.
+- **Wine** - runs Windows games and programs under a compatibility layer.
+- **Wine Steam** - runs the Steam Windows client under Wine.
+- **Yuzu** - another Nintendo Switch emulator.
+- **ZDoom** - the ZDoom DOOM Game Engine.
+
+**Services**
+
+Enable or disable all of your Game Source integration here. Any sources enabled will appear under the **Sources** menu on the left-hand side of the client.
+
+**Hardware information**
+
+Provides an output of your current system’s hardware, including **System**, **CPU**, **Memory** and **Graphics**. It is very useful if you need to troubleshoot performance issues with games.
+
+**Global options**
+
+Each of the options has already been explained in the **Manual Game Installation** section, however the difference is that each of the options here affects all of the games in your Lutris library, unless a game’s Runner options has been specifically changed.
+
+## Context Menu
+
+Right-clicking on any installed game in Lutris, provides the user with these options:
+- **Play** – runs the game.
+- **Show Logs** - opens a separate window and is useful for finding out why particular games are not running.
```

Comment:
Well, it won't output anything useful unless you enable logging. I would describe it as "useful for debugging purposes".